modules/ROOT/pages/scatter-gather-concept.adoc

```diff
@@ -19,6 +19,34 @@ To illustrate how this works, let’s look at two cases. In the first case, the
 
 In the second case, an error occurs in a route. This route either does not contain a Try scope or the error is of a type that the Try scope cannot handle. Although the other routes all complete, the one failed route causes the Scatter-Gather component to throw a `MULE.COMPOSITE ERROR` error, which the flow passes to the Error Handler component that you have configured for your flow. The error-handling event processors are able to process the Mule events from the completed routes.
 
+Example of handling this errors: 
+[source,xml,linenums]
+----
+<flow name="errorHandler">
+    <scatter-gather>
+        <route>
+            <raise-error type="APP:MYERROR"/>
+        </route>
+        <route>
+            <set-payload value="apple"/>
+        </route>
+    </scatter-gather>
+    <error-handler>
+        <on-error-continue type="COMPOSITE_ROUTING">
+            <!-- This will have the error thrown by the above processor -->
+            <logger level="WARN" message="#[error.errorMessage.payload.failures['0']]"/>
+            <!-- This will be a null value -->
+            <logger level="WARN" message="#[error.errorMessage.payload.failures['1']]"/>
+
+            <!-- This will be a null value -->
+            <logger level="WARN" message="#[error.errorMessage.payload.results['0']]"/>
+            <!-- This will have the result of this (correctly executed) route -->
```

Comment:
`This will have the result of this (correctly executed) route` -> `This will have the result of the second (correctly executed) route`

Reply:
```suggestion
            <!-- This will have the result of the second (correctly executed) route -->
```